app/views/groups/new.html.erb

```diff
@@ -0,0 +1,31 @@
+  <%= form_with(model: @group, url: user_groups_path) do |f| %>
+
+  <div class="nav-header">
+    <div>
+      <%= link_to  :back do %>
+        <%= image_tag "back.svg", style: 'height:10px; width:auto;' %>
+      <% end  %>
+    </div>
+    <h4 class="text-gray-4"> NEW CATTEGORY</h4>
+    <div class="actions">
+      <%=  f.submit ('NEXT'), class:" bg-light nav-right-action text-gray-4" %>
+    </div>
+  </div>
+
+    <div class= "form-group-custom" >
+      <%= f.text_field :name,   class:"form-control-custom", placeholder: "Enter category name", autofocus: true %>
```

Comment:
- [Optional] It will be nice to display an error message to let the user knows that this field should not be empty.